docs/changehistory/NextVersion.md

```diff
@@ -609,16 +609,26 @@ These methods were previously synchronous and are now async:
 - [InteractiveTool.onSuspend]($frontend)
 - [InteractiveTool.onUnsuspend]($frontend)
 
-## `NodeKey` in `@itwin/presentation-common`
+## Changes to `@itwin/presentation-common`
+
+### `NodeKey`
 
 The [NodeKey]($presentation-common) object contains a `pathFromRoot` attribute which can be used to uniquely identify a node in a hierarchy. In addition, the attribute is stable - the value for the same node is the same even when being created by different backends, which allows it to be persisted and later be used to identify specific nodes.
 
 In `3.0` changes have been made that changed the way this attribute is calculated, which means the same node produced by pre-3.0 and 3.x versions of `imodeljs` will have keys with different `pathFromRoot` value. To help identify the version of `NodeKey` a new `version` attribute has been added, with `undefined` or `1` being assigned to keys produced by pre-3.0 and `2` being assigned to keys produced by `3.x` versions of imodeljs. In addition, a new [NodeKey.equals]($presentation-common) function has been added to help with the equality checking of node keys, taking their version into account.
 
-## `KeySetJSON` in `@itwin/presentation-common`
+### `KeySetJSON`
 
 The format of [KeySetJSON]($presentation-common) has been changed to reduce its size. Instead of containing an array of instance IDs it now contains a single compressed IDs string. See [CompressedId64Set]($core-bentley) for more details about compressing IDs.
 
+### Changes to presentation rule specifications
+
+- [MultiSchemaClassesSpecification]($presentation-common) now contains [MultiSchemaClassesSpecification.arePolymorphic]($presentation-common) optional field to indicate if the specified classes should be handled polymorphically.
+- [ContentInstancesOfSpecificClassesSpecification.handleInstancesPolymorphically]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification.arePolymorphic]($presentation-common) have been deprecated.
+- New fields [ContentInstancesOfSpecificClassesSpecification.excludedClasses]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification.excludedClasses]($presentation-common) are now used to specify excluded instance classes.
+- [ContentInstancesOfSpecificClassesSpecification.handleInstancesPolymorphically]($presentation-common) is now used as default polymorphism value for [ContentInstancesOfSpecificClassesSpecification.classes]($presentation-common) and [ContentInstancesOfSpecificClassesSpecification.excludedClasses]($presentation-common).
+- [InstanceNodesOfSpecificClassesSpecification.arePolymorphic]($presentation-common) is now used as default polymorphism value for [InstanceNodesOfSpecificClassesSpecification.classes]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification.excludedClasses]($presentation-common).
```

Comment:
```suggestion
- Added ability to specify polymorphism at class level rather than specification level. 
  
  Previously polymorphism was specified at specification level using [ContentInstancesOfSpecificClassesSpecification.handleInstancesPolymorphically]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification.arePolymorphic]($presentation-common) flags. They're now deprecated in favor of the new [MultiSchemaClassesSpecification.arePolymorphic]($presentation-common) attribute and act as default values if the new attribute is not specified.

  The change allows [ContentInstancesOfSpecificClassesSpecification]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification]($presentation-common) specify multiple classes with different polymorphism, if necessary.

- Added ability to exclude some classes when creating content and hierarchies.

  New optional attributes [ContentInstancesOfSpecificClassesSpecification.excludedClasses]($presentation-common) and [InstanceNodesOfSpecificClassesSpecification.excludedClasses]($presentation-common) have been added to specify classes that could be excluded. This provides a convenient way describe what class instances should be excluded from content or hierarchy, e.g. `give me all bis.Element instances except bis.GeometricElement`.
```